runware/base.py

```diff
@@ -169,6 +170,12 @@ async def imageInference(
             await self.ensureConnection()
             control_net_data: List[IControlNetWithUUID] = []
 
+            if requestImage.maskImage:
+                if not requestImage.maskImage.startswith("http"):
```

Comment:
if not os.path.exists check maybe nice for user errors that read well / explain how to fix the issue